src/frequenz/channels/_select.py

```diff
@@ -228,16 +228,9 @@ def exception(self) -> Exception | None:
         """
         return self._exception
 
+    @property
     def was_stopped(self) -> bool:
```

Comment:
`Selected.was_stopped` feels a bit ambiguous to me.  Might be worth renaming to `Selected.receiver_stopped`.

Reply:
I find it clear enough but I'm also OK to rename it. Will wait a bit in case anyone else wants to chime in.

Reply:
I agree the suggested name is more precise. Even though  the new documentation will clear up any ambiguity I'd also suggest to rename it

Reply:
OK, I was doing the change but then wondered what about `exception` and `message`, should they also be `receiver_exception` and `receiver_message`? I'm not super convinced about introducing an inconsistency where some stuff in `Selected` has a `receiver_` prefix and some not, but also find a bit overkill to prepend everything with `receiver_` as it is pretty clear to me that `Selected` is wrapping a receiver IMHO.

Reply:
>I was doing the change but then wondered what about exception and message, should they also be receiver_exception and receiver_message? 

Fair point. No, i don't think so. Considering those other properties then I'd say it's fine to keep the existent name `was_stopped` without prepending `receiver_`

Reply:
@shsms ?

Reply:
The other fields don't need the `receiver_` prefix, because we don't have a `was_exception` or `was_message` to begin with.

Reply:
I agree that `was_` is probably unnecessary, `stopped` is already a binary state, I think I used it because it read more nicely with `if selected.was_stopped` than `if selected.stopped`. Also I think the past tense is still correct. It was stopped when it was used to receive, we don't know if it got somehow restarted since then.

> And it doesn't even qualify that it should be checked within a type narrowed scope of `selected_from`. It feels like it could be anything.

I didn't get this.

> The other fields don't need the `receiver_` prefix, because we don't have a `was_exception` or `was_message` to begin with.

I don't get the difference. `exception` and `message` are actual objects, not binary states, so it doesn't make sense to have a `was_`. I thought you suggested the prefix to indicate that it's talking about the receiver, not the `Selected` object.

In any case, to wrap up, if you are not completely opposed to it and there is no consensus on a better choice, I'd say we just keep the status quo to be able to move forward.

Reply:
I will merge this PR, as it is really a separate issue if we want to rename this or not. Feel free to create a issue for it @shsms and we can keep discussing it there.